tools/ci_build/build.py

```diff
@@ -576,6 +576,10 @@ def run_server_tests(build_dir, configs):
     for config in configs:
         config_build_dir = get_config_build_dir(build_dir, config)
         server_app_path = os.path.join(config_build_dir, 'onnxruntime_server')
+        if is_windows():
+            server_app_path = os.path.join(config_build_dir, config, 'onnxruntime_hosting.exe')
+        else:
+            server_app_path = os.path.join(config_build_dir, 'onnxruntime_hosting')
         server_test_folder = os.path.join(config_build_dir, 'server_test')
         server_test_data_folder = os.path.join(os.path.join(config_build_dir, 'testdata'), 'server')
```

Comment:
This one need to be double `Debug` in the path for Windows as well.

Reply:
This *is* double `Debug`